library/std/src/prelude/mod.rs

```diff
@@ -28,53 +28,53 @@
 //! The current version of the prelude (version 1) lives in
 //! [`std::prelude::v1`], and re-exports the following:
 //!
-//! * [`std::marker`]::{[`Copy`], [`Send`], [`Sized`], [`Sync`], [`Unpin`]}:
+//! * <code>[std::marker]::{[Copy], [Send], [Sized], [Sync], [Unpin]}</code>,
```

Comment:
Are we sure that Markdown will be rendered inside an HTML tag? I know that e.g. Jekyll will not render Markdown inside an HTML tag.

Reply:
Are you asking about anything in particular other than Rustdoc HTML output? Because that one works fine; I tested it and included screenshots.